src/main/kotlin/com/openlattice/launchpad/configuration/IntegrationRunner.kt

```diff
@@ -150,7 +149,7 @@ class IntegrationRunner {
             try {
                 ds.write()
                         .option("batchsize", destination.batchSize.toLong())
```

Comment:
is this one "batchsize" or "batchSize" (based on LaunchpadConstant below) ? Or does it not matter?